src/clib/grackle_chemistry_data.h

```diff
@@ -136,6 +136,21 @@ typedef struct
   /* flag for Wolcott-Green+ 2011 H2 self-shielding */
```

Comment:
Is this just a 0 or 1 thing (I know that this is not new code but maybe take this chance to clarify the behavior)?

Reply:
Yes this seems to be a 0 or 1 setting, will add a note for clarification.

Reply:
It can be 0,1,2 or 3. The different values correspond to the length scales used for the calculation of the H2 column density (https://grackle.readthedocs.io/en/latest/Parameters.html if you are interested in specifics). I will add a brief note of this to the code for clarification.